deploy/kubeturbo-operator/certified-operator-automation-1.py

```diff
@@ -0,0 +1,178 @@
+if local_repo_path is None:
+    local_repo_path = input('Enter repository local path: ')
+#PR parameters
+base_repo_name = 'certified-operators'
+base_repo_owner = 'redhat-openshift-ecosystem'
+origin_repo_name = 'certified-operators'
+origin_repo_owner = 'turbodeploy'
+#Git access token
+access_token = os.getenv('ACCESS_TOKEN')
+if access_token is None:
+    access_token = input('Enter turbodeploy access token: ')
+repo_url = f'https://turbodeploy:{access_token}@github.com/turbodeploy/certified-operators.git'
+
+# Clone the repository to a local directory
+git.Repo.clone_from(repo_url, local_repo_path)
+
+#To keep the code always in sync with base repository of redhat-openshift-ecosystem/certified-operators
+#before releasing any new operator
+# Add the base repository as a remote
+repo = git.Repo(local_repo_path)
+base_url = f'https://github.com/{base_repo_owner}/{base_repo_name}.git'
+repo.create_remote('base', base_url)
+# Fetch the changes from the base repository
+base = repo.remote(name='base')
+base.fetch()
+# Merge the changes into the local main branch
+repo.git.merge('base/main')
+# Push the changes to the forked repository
+origin = repo.remote(name='origin')
+origin.push()
+
+# Create Operator version folder
+operator_name = input('Enter the name of certified operator to release (kubeturbo or prometurbo):')
+operator_path = f'/operators/{operator_name}-certified'
+operator_dir_path = local_repo_path + operator_path
+operator_release_version = input(f'Enter the version of the {operator_name} certified operator to release: ')
+if not re.match(r'^\d+\.\d+\.\d+(-beta\.\d+)?$', operator_release_version):
+        raise ValueError('Invalid operator version format. It allows the format of X.Y.Z,or X.Y.Z-beta.N where X, Y, Zand N are digits.')
+os.chdir(operator_dir_path)
+if os.path.exists(operator_release_version):
+    raise ValueError('The version entered already exists, cannot release a duplicate version')    
+os.mkdir(operator_release_version)
+print(f'{operator_name} operator version {operator_release_version} directory created successfully.')
+
+# Copy files form user input operator version into new operator version folder
+operator_dir_to_copy = input(f'Enter the {operator_name} operator version to copy from: ')
+for filename in os.listdir(operator_dir_to_copy):
+    source_path = os.path.join(operator_dir_to_copy, filename)
+    if os.path.isdir(source_path):
+        shutil.copytree(source_path, os.path.join(operator_release_version, filename))
+    else:
+        shutil.copy(source_path, operator_release_version)
+print(f'{operator_name} operator version {operator_dir_to_copy} files copied successfully into {operator_release_version}.')
+
+
+client = docker.from_env()
+#To get regex pattern for the digest key based on operator
+#'icr.io/cpopen/kubeturbo(or prometurbo)-operator@sha256: xxxxx' 
+if operator_name == 'kubeturbo':
+ digest_regex = r'(icr.io/cpopen/kubeturbo-operator@sha256:[a-f0-9]{64})'
```

Comment:
Let's put regex in a variable and move these variable on top into one block. So that it will be easier to understand the purpose and update later on.